### PR TITLE
Feature/messages

### PR DESCRIPTION
**Pull request by bezidejni**
Need a quick code review, if everything looks ok, please merge. @natea @littleq0903

### CODE REVIEW

**Comment by natea** [comment]
cool, I'll take a look at this pull request this afternoon.

On Mon, Aug 12, 2013 at 11:49 AM, Filip Jukić notifications@github.comwrote:

> Need a quick code review, if everything looks ok, please merge. @nateahttps://github.com/natea
> 
> ## @littleq0903 https://github.com/littleq0903
> 
> You can merge this Pull Request by running
> 
>   git pull https://github.com/appsembler/mediathread feature/messages
> 
> Or view, comment on, or merge it at:
> 
>   https://github.com/appsembler/mediathread/pull/3
> Commit Summary
> - added django messaging bar to the template
> - Merge branch 'refs/heads/appsembler_develop' into feature/messages
> - added messaging to registration/invite views
> - Merge branch 'refs/heads/appsembler_develop' into feature/messages
> - added messaging settings
> - Merge branch 'refs/heads/appsembler_develop' into feature/messages
> - fixed a typo
> 
> File Changes
> - _M_ media/css/mediathread.csshttps://github.com/appsembler/mediathread/pull/3/files#diff-0(26)
> - _A_ media/img/icons/icon_alert.gifhttps://github.com/appsembler/mediathread/pull/3/files#diff-1(0)
> - _A_ media/img/icons/icon_error.gifhttps://github.com/appsembler/mediathread/pull/3/files#diff-2(0)
> - _A_ media/img/icons/icon_success.gifhttps://github.com/appsembler/mediathread/pull/3/files#diff-3(0)
> - _M_ mediathread/course/views.pyhttps://github.com/appsembler/mediathread/pull/3/files#diff-4(6)
> - _M_ mediathread/settings_shared.pyhttps://github.com/appsembler/mediathread/pull/3/files#diff-5(2)
> - _M_ mediathread/templates/base.htmlhttps://github.com/appsembler/mediathread/pull/3/files#diff-6(7)
> - _M_ mediathread/user_accounts/views.pyhttps://github.com/appsembler/mediathread/pull/3/files#diff-7(11)
> 
> Patch Links:
> - https://github.com/appsembler/mediathread/pull/3.patch
> - https://github.com/appsembler/mediathread/pull/3.diff

## 

nate@appsembler.com
+1 (617) 517-4953
http://twitter.com/natea | http://linkedin.com/in/natea

File: mediathread/course/views.py [hunk 1]
```diff
@@ -37,7 +38,8 @@ def form_valid(self, form):
         course.add_member(self.request.user, faculty=True)
 
         self.request.session['ccnmtl.courseaffils.course'] = course.course
-
+        messages.success(self.request,
+                         "You've successfully created a new course: {0}".format(course_title))
```
Comment (natea) [inline mediathread/course/views.py]: Should this have a fail_silently=True like the other two messages?

File: mediathread/user_accounts/views.py [hunk 2]
```diff
@@ -142,6 +143,10 @@ def form_valid(self, form):
                     invitor_email=form.cleaned_data['email_from'],
                     message=linebreaks(form.cleaned_data['message']),
                 )
+        student_count = len(emails)
+        messages.success(self.request,
+                         "You've successfully invited {0} students.".format(student_count),
+                         fail_silently=True)
```
Comment (natea) [inline mediathread/user_accounts/views.py:151]: If you only invite 1 student, it says, "You have successfully invited 1 students." It should say, "1 student".
screen shot 2013-08-12 at 1 08 52 pm

File: mediathread/user_accounts/views.py [hunk 3]
```diff
@@ -36,7 +37,7 @@ def post(self, *args, **kwargs):
         email_address = self.get_object().email_address
         user_to_login = User.objects.get(email=email_address.email)
         login_user(self.request, user_to_login)
-
+        messages.success(self.request, "You've successfully activated your account.", fail_silently=True)
```
Comment (natea) [inline mediathread/user_accounts/views.py:41]: I noticed that there is a notice "Successfully signed in as natejaune." when you signin, but it only appears on the course page. If I belong to more than one course, it takes me to the "Select a course" page, and no message is displayed.

Comment (bezidejni) [inline mediathread/user_accounts/views.py:41]: That's mediathread or allauth default behaviour, I added messages only for our newly added registration/invitation view. I can take a look at that but I guess it isn't a priority at the moment?

File: mediathread/course/views.py (hunk 1 above)
Comment (bezidejni) [inline mediathread/course/views.py]: Fixed.

File: mediathread/user_accounts/views.py (hunk 2 above)
Comment (bezidejni) [inline mediathread/user_accounts/views.py:151]: Fixed.

File: mediathread/user_accounts/views.py (hunk 3 above)
Comment (natea) [inline mediathread/user_accounts/views.py:41]: Yeah, not a big deal right now. I think this one is ready to merge.

**Pull request merged**